### PR TITLE
K8s migration

### PR DESCRIPTION
**Pull request by deelawn**
All the changes needed to migrate to k8s.

### CODE REVIEW

File: helm/celo-indexer/values.yaml [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+            SERVER_PORT: 8081
+            DEFAULT_BATCH_SIZE: 100
+            SERVER_METRIC_ADDR: "0.0.0.0:8090"
+          command:
+            - /app/celo-indexer
+            - -cmd=server
+          livenessProbe:
+            httpGet:
+              path: /liveness
+              port: 8090
```
Comment (christophermanning) [inline helm/celo-indexer/values.yaml:80]: Technically, all these livenessProbe and readinessProbe should be calling an endpoint on 8081; refer to the `health` package in `indexing-engine` or the `polkadothub-indexer` for examples. As-is, it's calling the metrics server handler.

Comment (deelawn) [reply helm/celo-indexer/values.yaml:80]: I've looked at some of the other liveness/readiness probes that are defined in other services and I'm not sure they are functionally different from how it works here.  For example, this seems to be the common pattern with referring to the monitor code from the indexing-engine:
```
monitor := &health.Monitor{}
go monitor.RunChecks(ctx, cfg.HealthCheckInterval)
monitor.AttachHttp(mux)
```
But if there are no probes defined for the monitor, it just returns a 200 when the health/liveness/readiness endpoints are called; so hitting these liveness and readiness endpoints should yield the same results as hitting an endpoint that is returning metrics.

I see a postgres probe defined in a subpackage of `health` in the indexing-engine that could be added to the monitor to make it actually do something.  But I also see some code commented out in the monitor's liveness check. @lukanus , should the postgres probe be added to these monitors?  Any information about why the liveness code was commented out?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -1,63 +1,138 @@
...
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
 
       - name: Set ref vars
         id: refs
         run: |
           echo ::set-output name=SOURCE_NAME::${GITHUB_REF#refs/*/}
           echo ::set-output name=SOURCE_BRANCH::${GITHUB_REF#refs/heads/}
           echo ::set-output name=SOURCE_TAG::${GITHUB_REF#refs/tags/}
+      - name: Configure AWS Credentials
```
Comment (lukanus) [inline .github/workflows/ci.yaml:31]: Out of curiosity, why do we need this in CI?

File: Dockerfile [hunk 3]
```diff
@@ -20,17 +20,33 @@ RUN \
...
+
 # ------------------------------------------------------------------------------
 # Target Image
 # ------------------------------------------------------------------------------
 FROM alpine:3.10 AS release
 
+ENV GLIBC_REPO=https://github.com/sgerrand/alpine-pkg-glibc
+ENV GLIBC_VERSION=2.30-r0
+
+RUN set -ex && \
```
Comment (lukanus) [inline Dockerfile:32]: It's for local m1 ?

File: helm/celo-indexer/values.yaml (hunk 1 above)
Comment (lukanus) [reply helm/celo-indexer/values.yaml:80]: This readiness/liveness probe is a bit cheat. it works fine and fills it's purpose. 
So both are fine-ish, if there is any interface that returns 200, and readiness check happens after blocking database connection check. 
I created this package so we can develop code checking database in one place as @deelawn mentioned. This way health checks (that also return information for metrics) can also check the state of service. The only thing that's checked right now is ping and size of database, if configured

File: Dockerfile (hunk 3 above)
Comment (deelawn) [reply Dockerfile:32]: No, I don't have an m1.  The error was actually on deployment:
https://github.com/figment-networks/celo-indexer/runs/4881238315?check_suite_focus=true

**Review by lukanus — APPROVED** [review]
👍  great job.
There is one thing it would be great to add.
https://github.com/figment-networks/polkadothub-indexer/commit/3904c36a4a928eae899d6e6e33aacc933b746251#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4R29
this excludes following path to be logged on datadog. I see health is still logged. 
Also not in this PR, but we can maybe set the pipeline metrics in datadog and just hide all the logs from pipeline processing? 
Current polkadot has the same problem and near/oasis soon will have the same

File: Dockerfile (hunk 3 above)
Comment (deelawn) [reply Dockerfile]: Now using ubuntu 20.04 image. Added the following to fix a certificate issue:
```
RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates
```

**Comment by deelawn** [comment]
> 👍 great job. There is one thing it would be great to add. [figment-networks/polkadothub-indexer@3904c36#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4R29](https://github.com/figment-networks/polkadothub-indexer/commit/3904c36a4a928eae899d6e6e33aacc933b746251#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4R29) this excludes following path to be logged on datadog. I see health is still logged. Also not in this PR, but we can maybe set the pipeline metrics in datadog and just hide all the logs from pipeline processing? Current polkadot has the same problem and near/oasis soon will have the same

I made most of the unnecessary logs "debug" level and removed verbose logger from the cron job runner.

**Pull request merged**